datadog_checks_dev/datadog_checks/dev/tooling/templates/integration/check/{check_name}/manifest.json

```
@@ -2,23 +2,40 @@
   "manifest_version": "2.0.0",
   "app_uuid": "{app_uuid}",
   "app_id": "{integration_id}",
-  "display_on_public_website": false,
+  "display_on_public_website": true,
   "tile": {{
     "overview": "README.md#Overview",
     "configuration": "README.md#Setup",
     "support": "README.md#Support",
     "changelog": "CHANGELOG.md",
-    "description": "",
+    "description": "<fill in>",
```

```suggestion
    "description": "<FILL IN>",
```
Can you use all caps for these?